velox/expression/CastExpr-inl.h

```diff
@@ -727,34 +727,6 @@ void CastExpr::applyCastPrimitives(
       });
     }
   }
-
-  // If we're converting to a TIMESTAMP, check if we need to adjust the
```

Comment:
this was the wrong place to do this adjustment as at this point we already lost the context of what was found when parsing the string. This logic is now done in the PrestoCastHook